tsdb/importer/import_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package importer
+
+import (
+	"os"
+	"testing"
+
+	"github.com/prometheus/prometheus/util/testutil"
+)
+
+func TestCleanDir(t *testing.T) {
+	t.Cleanup(func() {
+		testutil.Ok(t, os.RemoveAll("/Users/atibhi/Desktop/prometheus/tmpDir"))
+	})
+}
+
+// func TestImport(t *testing.T) {
```

Comment:
Let's not commit commented code (:

Reply:
yep, since this is WIP I committed for now. After writing the tests, will remove it.